tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -461,35 +461,62 @@ install_initrd_confidential() {
 	install_initrd "confidential"
 }
 
-#Instal NVIDIA GPU image
+#Install Mariner guest initrd
+install_initrd_mariner() {
+	install_initrd "mariner"
+}
```

Comment:
This was removed a few commits ago, and I think you mistakenly re-added it as part of a rebase.

Reply:
Removed.